mappings/net/minecraft/entity/Mountable.mapping

```diff
@@ -1,4 +1,5 @@
-CLASS net/minecraft/class_5146
+CLASS net/minecraft/class_5146 net/minecraft/entity/Mountable
```

Comment:
This class is already mapped. This change is wrong, as it renames from a nonexistent file. (Should rename from `Saddleable`)

Reply:
This class was most definitely not mapped when I was working on this PR.

Reply:
> 
> 
> This class was most definitely not mapped when I was working on this PR.

Well, this isn't an excuse for you not to fix things like merge conflicts. There will be a format error, so please merge the current 20w15a branch and delete the saddleable mapping to make the pr right. I am not asking you to take the `Saddleable` name, read my last comment again.

Reply:
It should be named Saddleable, or something relating to saddles. It's ONLY implemented by entities that can have a saddle, and is only used by the saddle item and saddle dispenser behaviour.

Reply:
> 
> 
> > This class was most definitely not mapped when I was working on this PR.
> 
> This isn't an excuse for you not to fix merge conflicts. There will be a format error, so please merge the current 20w15a branch and delete the `Saddleable` mapping to make the pr right. I am not asking you to take the `Saddleable` name, read my last comment again.

Why didn't you say so? Sure, I'll fix the conflict.